src/rsz/src/EstimateWireParasitics.cc

```diff
@@ -391,6 +397,8 @@ void Resizer::ensureWireParasitic(const Pin* drvr_pin, const Net* net)
         parasitics_invalid_.erase(net);
         break;
       }
+      case ParasiticsSrc::detailed_routing:
+        break;
```

Comment:
no action?

Reply:
Similar comment here. The idea is that the parasitics are already loaded in sta.